index.html

```diff
@@ -217,5 +217,53 @@ <h2 class="skill-name">Skills</h2>
 			</li>
 		</ul>
 	</section>
+	<section  class="contact-container" id="footer-container-id">
+		<div class="contact-form">
+			<h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>		
```

Comment:
- [ ] As suggested by the previous reviewer, please make sure that you follow the figma template according to [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). Following is a quote for a related requirement. 🔽 The correct color on figma template is `color: #172B4D` 

- [ ] As suggested by the reviewer [here](https://github.com/yosaddis/Portfolio/pull/4/files#r1072948825). Please make sure to stick to the fonts also. I know you can do that 💪

  > _You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)._

## **_Your design previously 👇_**
![image](https://user-images.githubusercontent.com/106119105/213324182-a20c0620-8938-4308-8671-3a467dafb44a.png)

## **_Your design now 👇_**
![image](https://user-images.githubusercontent.com/106119105/213324200-a7fd93d8-419b-4981-b457-855d34729989.png)

## **_Figma design 👇_**
![image](https://user-images.githubusercontent.com/106119105/213324265-d50d6374-5a74-473b-8365-6e2f84697007.png)